src/Api/index.js

```diff
@@ -33,6 +33,16 @@ const apis = {
 	accomodations() {
 		return axios.get(`${BASE_URL}accommodations`);
 	},
+	async getAllAccomodations() {
+		const accommodations = await axios.get(`${BASE_URL}accommodations`);
+		return accommodations;
+	},
+	async singleAccomodation(id) {
+		return await axios.get(`${BASE_URL}accommodations/${id}`);
```

Comment:
great job @crispy but your using wastelessly await and return statement its better you remove async function and leave it as a row function

Reply:
you can idealy see as in examples above